go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/google/go-cmp v0.7.0
 	github.com/google/go-github/v69 v69.2.0
 	github.com/googleapis/gax-go/v2 v2.14.2
+	github.com/stretchr/testify v1.10.0
```

Comment:
Don't use this package, or any test helper package.

Reply:
https://go.dev/wiki/TestComments#assert-libraries

Reply:
Done.